blog.html

```diff
@@ -139,15 +141,15 @@ <h3>Kafka Broker, Controller, Producer, Consumer and Admin Client</h3>
                         <br>
                         KIP-848 and KIP-932 introduce new group types, consumer and share respectively. The Admin Client
                         API has limitations when dealing with newer group types, sometimes returning errors or incorrectly
-                        stating that groups don't exist. To address this challenge a new command-line tool, kafka-groups.sh,
-                        and updates to existing command line tools, kafka-consumer-groups.sh and kafka-share-groups.sh,
+                        stating that groups don't exist. To address this challenge, a new command-line tool, kafka-groups.sh,
+                        and updates to existing command line tools, <code>kafka-consumer-groups.sh</code> and kafka-share-groups.sh,
```

Comment:
It seems we missed this one:
```suggestion
                        and updates to existing command line tools, <code>kafka-consumer-groups.sh</code> and <code>kafka-share-groups.sh</code>,
```